src/CodeGen_LLVM.h

```diff
@@ -84,6 +84,14 @@ class CodeGen_LLVM : public IRVisitor {
     virtual void compile_buffer(const Buffer &buffer);
     // @}
 
+    /** Helper functions for compiling Halide functions to llvm
```

Comment:
Please document that these are guaranteed to be called in a matched pair with compile_func in-between.